test/benchmark/redis/set_100k_conc20_rps/workload_config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "workload_type": "host_guest"
```

Comment:
Since the directory already includes a `host.sh` script, the benchmark framework should be able to figure out that this workload is host-guest, right? So I think this configuration file is unnecessary.